### PR TITLE
Daemonsets no helm

### PR DESCRIPTION
**Pull request by Arau**

### CODE REVIEW

File: _docs/install/kubernetes/index.md [hunk 1]
```diff
@@ -31,13 +33,17 @@ of the kubelets. This will be [fixed](https://github.com/kubernetes/kubernetes/p
...
+
 ## Install with Helm
 
-Firstly, [install Helm](https://docs.helm.sh/using_helm).
+Firstly, [install Helm](https://docs.helm.sh/using_helm). If you are running a rbac enabled cluster, you will need to create a [service account](https://github.com/kubernetes/helm/blob/master/docs/rbac.md).
 
 The StorageOS helm chart triggers an [init container](https://github.com/storageos/init) to enable required kernel configuration on your host. If that procedure doesn't finish successfully, StorageOS container will not start.
 
-If that is your case checkout the [os support](/docs/reference/os_support) page as you can manually prepare the settings for the kernel.
+If that is your case, checkout the [os support](/docs/reference/os_support) page as you can manually prepare the settings for the kernel.
```
Comment (oicheryl) [inline _docs/install/kubernetes/index.md]: "If that is your case, checkout" => "If that is the case, check out"

File: _docs/install/kubernetes/index.md [hunk 2]
```diff
@@ -57,3 +63,246 @@ $ helm delete --purge my-release
...
+    parameters:
+      pool: default
+      description: Kubernetes volume
+      fsType: ext4
+      adminSecretNamespace: default
+      adminSecretName: storageos-api
+    END
+    ```
+
+    StorageOS uses a secret to discover the API endpoint of its workers. `adminSecretNamespace` have to be set according to the namespace created previously. Equally important is that
```
Comment (oicheryl) [inline _docs/install/kubernetes/index.md]: StorageOS uses a secret to discover the API endpoint of its workers. `adminSecretNamespace` must be set to the specified namespace, and the service account needs to have granted permissions to StorageOS pods to have access to that secret.

(and same below)

File: _docs/install/kubernetes/index.md [hunk 3]
```diff
@@ -57,3 +63,246 @@ $ helm delete --purge my-release
...
+    the service account grants permissions to StorageOS pods to have access to that secret.
+
+
+1. Create DaemonSet
+
+    StorageOS server pod runs as a daemonset, therefore all instances of your cluster will have a StorageOS worker. In case that you want to limit the amount of nodes in the StorageOS cluster
```
Comment (oicheryl) [inline _docs/install/kubernetes/index.md]: instances of your cluster => nodes in your cluster

Comment (oicheryl) [reply _docs/install/kubernetes/index.md]: In case that => If

File: _docs/install/kubernetes/index.md [hunk 4]
```diff
@@ -57,3 +63,246 @@ $ helm delete --purge my-release
...
+    or make sure which Kubernetes nodes run storage workloads, you can define `spec.nodeSelector` based on node tags. For more information check [here](https://kubernetes.io/docs/concepts/configuration/assign-pod-node).
```
Comment (oicheryl) [inline _docs/install/kubernetes/index.md]: make sure => enforce

File: _docs/install/openshift/index.md [hunk 5]
```diff
@@ -18,6 +18,8 @@ type to install to make devices usable from within containers.
 
 You will need an OpenShift 3.8+ cluster with Beta APIs enabled.
 
+1. Install [StorageOS CLI](/docs/reference/cli/index).
```
Comment (oicheryl) [inline _docs/install/openshift/index.md]: Use {%link %} links.

File: _docs/install/openshift/index.md [hunk 6]
```diff
@@ -52,3 +59,255 @@ $ helm delete --purge my-release
...
+    parameters:
+      pool: default
+      description: Kubernetes volume
+      fsType: ext4
+      adminSecretNamespace: default
+      adminSecretName: storageos-api
+    END
+    ```
+
+    StorageOS uses a secret to discover the API endpoint of its workers. `adminSecretNamespace` have to be set according to the namespace created previously. Equally important is that
```
Comment (oicheryl) [inline _docs/install/openshift/index.md]: See above wording.

File: _docs/install/kubernetes/index.md (hunk 1 above)
Comment (Arau) [reply _docs/install/kubernetes/index.md]: Done

File: _docs/install/kubernetes/index.md (hunk 2 above)
Comment (Arau) [reply _docs/install/kubernetes/index.md]: Done

File: _docs/install/kubernetes/index.md (hunk 3 above)
Comment (Arau) [reply _docs/install/kubernetes/index.md]: Done

File: _docs/install/kubernetes/index.md (hunk 4 above)
Comment (Arau) [reply _docs/install/kubernetes/index.md]: Done

File: _docs/install/openshift/index.md (hunk 5 above)
Comment (Arau) [reply _docs/install/openshift/index.md]: Done

File: _docs/install/openshift/index.md (hunk 6 above)
Comment (Arau) [reply _docs/install/openshift/index.md]: Done

**Pull request merged**